### PR TITLE
Prajna-patch

### PR DESCRIPTION
**Pull request by Prajnaaa12**
This Pull Request resolves Issue #2

Description:
In this Pull Request we have done the following additions/edits for the code:

Note: Delete all the text including this!
* Make sure you replace `#<IssueNumber>` with the particular Issue that you are referencing with!
* Make sure you describe the Issue properly and ask as many people as possible to review your Pull Request so that it can be merged as soon as possible!

### CODE REVIEW

File: Participant/helloworld.c [hunk 1]
```diff
@@ -0,0 +1,4 @@
+#include<stdio.h>
+void main()
+{
+printf("Hello World!");
```
Comment (salman-bhai) [inline Participant/helloworld.c]: Put up a closed bracket and return function for the C program as well!

**Comment by mishal23** [comment]
Any updates @Prajnaaa12 ?

**Comment by salman-bhai** [comment]
Are there any updates here or should we close the PR and label it as `spam`?

**Comment by Prajnaaa12** [comment]
I'll update soon.. please don't close...i'm bit busy now

**Comment by salman-bhai** [comment]
Alright!

**Comment by salman-bhai** [comment]
Hey change the directory!

**Comment by Prajnaaa12** [comment]
How to change it?

**Comment by mishal23** [comment]
Clone the **forked** repo, move the ```helloworld.c``` from Participant folder into the required directory.
Commit the new changes in this PR.

**Comment by Prajnaaa12** [comment]
Actually I can't find helloworld.c in Participant folder. Only 1 file with my details can be seen in that folder.

**Comment by Prajnaaa12** [comment]
Now I created a new pull request for the same issue by creating a new file in another directory. so, I think this pull request can be closed.

**Comment by Prajnaaa12** [comment]
I got the problem now..I was searching for participant folder in other branch but it was in patch-1 branch. Anyway I have created another pull request for same issue.

**Pull request closed without merging**